src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,59 @@
+import { useSearchParams } from 'react-router-dom';
+import { FilterCentury } from '../types/FilterCentury';
+import { FilterSex } from '../types/FilterSex';
+
+import cn from 'classnames';
+import { useCallback, useMemo } from 'react';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sexParam = useMemo(() => searchParams.get('sex') || '', [searchParams]);
+  const centutyParams = useMemo(
```

Comment:
There is a typo in the variable name `centutyParams`. It should be corrected to `centuryParams` to match the intended meaning and improve code readability.